selene/schema/policies/fields.py

```diff
@@ -101,9 +102,19 @@ class Policy(EntityObjectType):
     known_nvt_count = graphene.Int()
     predefined = graphene.Boolean()
 
-    families = graphene.List(PolicyFamily)
-    preferences = graphene.List(NvtPreference)
-    tasks = graphene.List(PolicyTask)
+    families = graphene.List(
+        PolicyFamily, description='List of NVT Families in this Policy'
+    )
+    nvt_preferences = graphene.List(
+        NvtPreference, description='List of NVT Preferences for this Policy'
+    )
+    scanner_preferences = graphene.List(
+        ScannerPreference,
+        description='List of Scanner Preferences for this Policy',
+    )
+    tasks = graphene.List(
+        PolicyTask, description='List of Tasks using this Policy'
+    )
```

Comment:
@sarahd93 shouldn't this be named audits like
```suggestion
    audits = graphene.List(
        PolicyAudit, description='List of Audits using this Policy'
    )
```

Reply:
Copy Paste. :/